api/src/config/index.ts

```diff
@@ -1,5 +1,5 @@
 import { config } from 'dotenv';
-config({ path: `.env.${process.env.NODE_ENV || 'development'}.local` });
+config();
```

Comment:
After thinking on this, I think let's consider undoing this. This will require you to have multiple different env files:
`.env.test.local`, `.env.development.local`, etc - this will be better because then you can setup your test database not to conflict with your development database, not to conflict with your production config.